includes/discovery/discovery-protocols.inc.php

```diff
@@ -117,17 +117,14 @@
     echo ' LLDP-MIB: ';
     $lldp_array  = snmpwalk_group($device, 'lldpRemTable', 'LLDP-MIB', 3);
     if (!empty($lldp_array)) {
-        $dot1d_array = snmpwalk_group($device, 'dot1dBasePortIfIndex', 'BRIDGE-MIB');
```

Comment:
How do you know that this won't break things?

Reply:
I don't, but that data doesn't include all ports, so I couldn't use it on the Cisco device I was trying to fix.  This needs testing against several devices.  On my devices, this worked out a lot better.

We could walk both and then we would have more data.

Reply:
Walking both would be better imho. I can't test this and we may not see many people testing it to be sure